Sources/FloatLabelRow.swift

```diff
@@ -352,68 +370,89 @@ public class ZipCodeFloatLabelCell: _FloatLabelCell<String>, CellType {
     }
 }
 
+// MARK: Rows with formatter
+open class _IntFloatRow: FloatLabelRow<IntFloatLabelCell> {
+    public required init(tag: String?) {
```

Comment:
Please add empty line after class declarations and before closing class declaration